apps/penxle.com/src/routes/(default)/me/settings/contentfilters/MutedSpaceModal.svelte

```diff
@@ -0,0 +1,118 @@
+<script lang="ts">
+  import { fragment, graphql } from '$glitch';
+  import { Button, Image, Modal } from '$lib/components';
+  import { toast } from '$lib/notification';
+  import type { MeSettingsContentFiltersPage_MutedSpaceModal_user } from '$glitch';
+
+  let _user: MeSettingsContentFiltersPage_MutedSpaceModal_user;
+  export { _user as $user };
+  export let open = false;
+
+  let mutedSpaces: typeof $user.mutedSpaces;
+
+  const setSpaces = () => {
+    mutedSpaces = $user.mutedSpaces;
+  };
+
+  $: if (open) {
+    setSpaces();
+  }
+
+  $: user = fragment(
+    _user,
+    graphql(`
+      fragment MeSettingsContentFiltersPage_MutedSpaceModal_user on User {
+        id
+
+        mutedSpaces {
+          id
+          name
+          description
+          muted
+
+          icon {
+            id
+            ...Image_image
+          }
+        }
+      }
+    `),
+  );
+
+  const muteSpace = graphql(`
+    mutation MeSettingsContentFiltersPage_MuteSpace_Mutation($input: MuteSpaceInput!) {
+      muteSpace(input: $input) {
+        id
+        muted
+      }
+    }
+  `);
+
+  const unmuteSpace = graphql(`
+    mutation MeSettingsContentFiltersPage_UnmuteSpace_Mutation($input: UnmuteSpaceInput!) {
+      unmuteSpace(input: $input) {
+        id
+        muted
+      }
+    }
+  `);
+
+  const onInput = (event: Event) => {
+    const { value } = event.currentTarget as HTMLInputElement;
```

Comment:
P3: type assertion 사용을 가능한 지양합시다! 입력 이벤트 핸들러 타입을 아래와 같이 직접 참조할 수 있어요! 

```suggestion
  const onInput: FormEventHandler<HTMLInputElement> = (event) => {
    const { value } = event.currentTarget;
```